python_ta/cfg/graph.py

```diff
@@ -2,8 +2,11 @@
 
 from typing import Any, Generator, List, Optional, Set, Tuple, Union
 
+import z3
 from astroid import Break, Continue, NodeNG, Raise, Return
 
+from ..transforms import ExprWrapper
```

Comment:
`ExprWrapper` is unused. I tried using it but since function definitions are not expressions, I had trouble figuring out how to apply what `ExprWrapper` can do without contorting and modifying its original functionality.

Reply:
You can change the type for the `node` parameter in the initializer to `astroid.NodeNG` (matching the actual attribute annotation).

Then in the code below, you can use the `types` attribute of the instance of `ExprWrapper` instead of just a local variable `types`, and similarly use the `ExprWrapper.reduce` method and call it on the parameter names. You might need to add an extra case for `AssignName` nodes, I'm not sure.